devcontainer/files/scripts/start-celery.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+source $BRIDGE_SERVER_PATH/bridge-server-venv/bin/activate
+celery -A util.tasks worker -c=1 -E -B -l debug
```

Comment:
Should the celery beat process (i.e. the scheduler) also get started here ? Something along the line of:
`celery -A util.tasks beat`

Reply:
Yes, please include all things celery 🌵

Reply:
Gotcha. This change was made when I was setting the logging level and saw that the -B argument runs celery beat with the worker process. It's meant for dev environments, and allows one tab or process to be run instead of two.

Along the lines of mirroring production, I see that Celery can be run this way in Heroku: https://drdaeman.github.io/heroku-djcelery-example/

@Gzing - in our production bridge server, are we running a two dyno, three dyno, or add-on setup for celery? Thanks!